Remove btouch nullable warning.

The following warnings are showed during the build:
```
/Users/mandel/Xamarin/xamarin-macios/master/xamarin-macios/src/btouch.cs(50,16): warning CS8618: Non-nullable field 'outfile' must contain a non-null value when exiting constructor. Consider declaring the field as nullable. [/Users/mandel/Xamarin/xamarin-macios/master/xamarin-macios/src/bgen/bgen.csproj]
/Users/mandel/Xamarin/xamarin-macios/master/xamarin-macios/src/btouch.cs(52,9): warning CS8618: Non-nullable field 'baselibdll' must contain a non-null value when exiting constructor. Consider declaring the field as nullable. [/Users/mandel/Xamarin/xamarin-macios/master/xamarin-macios/src/bgen/bgen.csproj]
/Users/mandel/Xamarin/xamarin-macios/master/xamarin-macios/src/btouch.cs(53,9): warning CS8618: Non-nullable field 'attributedll' must contain a non-null value when exiting constructor. Consider declaring the field as nullable. [/Users/mandel/Xamarin/xamarin-macios/master/xamarin-macios/src/bgen/bgen.csproj]
/Users/mandel/Xamarin/xamarin-macios/master/xamarin-macios/src/btouch.cs(58,29): warning CS8618: Non-nullable field 'universe' must contain a non-null value when exiting constructor. Consider declaring the field as nullable. [/Users/mandel/Xamarin/xamarin-macios/master/xamarin-macios/src/bgen/bgen.csproj]
/Users/mandel/Xamarin/xamarin-macios/master/xamarin-macios/src/btouch.cs(60,20): warning CS8618: Non-nullable field 'Frameworks' must contain a non-null value when exiting constructor. Consider declaring the field as nullable. [/Users/mandel/Xamarin/xamarin-macios/master/xamarin-macios/src/bgen/bgen.csproj]
/Users/mandel/Xamarin/xamarin-macios/master/xamarin-macios/src/btouch.cs(61,26): warning CS8618: Non-nullable field 'AttributeManager' must contain a non-null value when exiting constructor. Consider declaring the field as nullable. [/Users/mandel/Xamarin/xamarin-macios/master/xamarin-macios/src/bgen/bgen.csproj]
```

--- a/src/btouch.cs
+++ b/src/btouch.cs
@@ -47,18 +47,18 @@ public class BindingTouch {
 	public PlatformName CurrentPlatform;
 	public bool BindThirdPartyLibrary = true;
 	public bool skipSystemDrawing;
-	public string outfile;
+	public string? outfile;
 
-	string baselibdll;
-	string attributedll;
+	string? baselibdll;
+	string? attributedll;
 
 	List<string> libs = new List<string> ();
 	List<string> references = new List<string> ();
 
-	public MetadataLoadContext universe;
+	public MetadataLoadContext? universe;
 	public TypeManager TypeManager = new TypeManager ();
-	public Frameworks Frameworks;
-	public AttributeManager AttributeManager;
+	public Frameworks? Frameworks;
+	public AttributeManager? AttributeManager;
 
 	readonly Dictionary<System.Type, Type> ikvm_type_lookup = new Dictionary<System.Type, Type> ();
 	internal Dictionary<System.Type, Type> IKVMTypeLookup {
@@ -98,7 +98,7 @@ public class BindingTouch {
 	string GetAttributeLibraryPath ()
 	{
 		if (!string.IsNullOrEmpty (attributedll))
-			return attributedll;
+			return attributedll!;
 
 		switch (CurrentPlatform) {
 		case PlatformName.iOS:

